src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -162,4 +162,89 @@
     <oldName>DSCore.Logic.Or</oldName>
     <newName>DSCore.Math.Or</newName>
   </priorNameHint>
+  <!-- NEW ADDITIONS -->
+  <!-- Many of these nodes are already remapped above.
+       Since these changes were introduced for 2.0 the
+       'newNames' can just be replaced if we don't need to support
+       any existing 2.0 graphs otherwise we need to remap twice-->
+  <priorNameHint>
```

Comment:
So I tried doing this 2 ways and neither seemed to work.

1) Add all new migrations to existing even though some would be attempting to migrate twice
2) Replacing the duplicate migration as outlined in the comment above

Both ways I still get the new dummy node error message we just created trying to open a 1.3 XML graph with these nodes in 2.0

Reply:
have you tried the also known as attribute yet?

Reply:
Yeah I tried using
```C#
    [AlsoKnownAs("DSCore.File.WriteText", "DSCore.IO.File.WriteText")]
    public class FileSystem : MigrationNode
    {
        [NodeMigration(version: "1.3.0.0")]
        public static NodeMigrationData Migrate_1300_to_2000(NodeMigrationData data)
        {
            NodeMigrationData migrationData = new NodeMigrationData(data.Document);
            XmlElement oldNode = data.MigratedNodes.ElementAt(0);

            XmlElement newNode = MigrationManager.CloneAndChangeName(
                // Existing migration from DSCore.File.WriteText
                // to DSCore.IO.File.WriteText (commented out) but
                // now is DSCore.IO.FileSystem.WriteText
                oldNode, "DSCore.IO.FileSystem.WriteText", "FileSytem.WriteText");

            migrationData.AppendNode(newNode);
            return migrationData;
        }
    }
```

 Where ` DSCore.IO.FileSystem.WriteText` is the new naming.  I tried a few different variations of this such as leaving the interim migration and removing it.  I also tried just migrating from the interim naming to the new naming since my test file was saved in the latest version of 1.3.  

`DSCore.File.WriteText` -> Original
`DSCore.IO.File.WriteText` -> Interim (Existing XML migration)
`DSCore.IO.FileSystem.WriteText` -> Latest

Am I using the version numbering correctly?  My test file was saved in 1.3.1.1736 and attempted migration was 2.0.0.3094 so I assumed specifying anything 1.3 and newer would be migrated.

Reply:
Added a breakpoint and I don't think this migration is ever getting invoked from `\src\Migrations\CoreNodes\dynBaseTypes.cs` when opening the 1.3 XML file

Reply:
the also known as attribute goes on the class itself, not on the migration - it should be used **Instead** of the migration.

Reply:
Ah okay, I was following [these](https://github.com/DynamoDS/Dynamo/blob/master/src/Migrations/CoreNodes/dynBaseTypes.cs#L75) existing migrations for Number Range and Number Sequence that use both the attribute and the migration for some reason?  

Tried adding the also known as attribute to the class itself and same result
```C#
[AlsoKnownAs("DSCore.File.WriteText", "DSCore.IO.File.WriteText")]
public static void WriteText(string filePath, string text)
{
    var fullpath = AbsolutePath(filePath);
     System.IO.File.WriteAllText(fullpath, text);
}
```